R/connect.R

```diff
@@ -59,7 +59,7 @@ connectClient <- function(service, authInfo) {
     },
 
     currentUser = function() {
-      handleResponse(GET(service, authInfo, "/users/current/"))
+      handleResponse(GET(service, authInfo, "/users/current"))
```

Comment:
I thought we redirected when there is no trailing slash (`trailingslash.go`). Is this something specific to the API or something new?